api/src/main/java/io/grpc/ServerInterceptors.java

```diff
@@ -187,13 +187,15 @@ public InputStream parse(final InputStream stream) {
       wrappedMethods.add(wrapMethod(definition, wrappedMethodDescriptor));
```

Comment:
`wrappedMethods` is never used, and this is causing the CI error.

Reply:
Look like the earlier for loop should not have been deleted and should occur after `serviceBuilder` is created (just like before).

Reply:
Sorry, not sure what happened--some patching problem on my end. Should look something like this:

```
    // Build the new service descriptor
    final ServiceDescriptor.Builder serviceDescriptorBuilder =
        ServiceDescriptor.newBuilder(serviceDef.getServiceDescriptor().getName())
            .setSchemaDescriptor(serviceDef.getServiceDescriptor().getSchemaDescriptor());
    // Create the new service definition.
    final ServerServiceDefinition.Builder serviceBuilder =
        ServerServiceDefinition.builder(serviceDescriptorBuilder.build());
    for (ServerMethodDefinition<?, ?> definition : wrappedMethods) {
      serviceBuilder.addMethod(definition);
    }
    return serviceBuilder.build();
```

Reply:
Applied fix.